src/backend/protocol.ts

```diff
@@ -10,8 +10,9 @@ import { z } from 'zod'
 import { windowIcon } from './constants/paths'
 import { Path } from './schemas'
 import { isCLINoGui } from './constants/environment'
+import { runnerMap } from 'backend/runners'
 
-const RUNNERS = z.enum(['legendary', 'gog', 'nile', 'sideload'])
+const RUNNERS = z.enum(Object.keys(runnerMap) as [Runner, ...Runner[]])
```

Comment:
This is a little awkward. Out of scope here, but we should look into making this a branded type with a refine instead then